frontend/src/components/SystemsTable/SystemTableContent.tsx

```diff
@@ -220,6 +247,50 @@ export function SystemTableContent({
     },
   };
 
+  const handleTableChange = (
```

Comment:
@lyuyangh, I try to use the `onFilter` property in `ColumnsType` definition to call the `onFilterChange` function to update the filtered system table, but I met some problems. 
* `onFilter: (value, record) => boolean`: reseting the filter doesn't set the value to be null. `onFilter` is not called.
* `<Table onChange={onChange} />` must have a `onChange` function so that the column's `onFilter` can be called. 

Therefore, in PR #528 , I still keep the `handleTableChange` function since Table's `onChange` property must have value and filters can get the reset null value directly.